lusidtools/cocoon/seed_sample_data.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+from lusidtools.cocoon.cocoon import load_from_data_frame
+from lusidtools.cocoon.utilities import load_json_file
+import logging
+
+logger = logging.getLogger()
+
+mappings = dict(load_json_file("config/seed_sample_data.json"))
+
+def seed_sample_data(api_factory, file_types, scope: str, transaction_source, mappings: dict = mappings):
+
+    """
+    Description:
+    This function allows users to seed their LUSID environment with some core data (e.g. instruments,
+    portfolios, and transaction) from one file.
+
+    :param lusid.utilities.ApiClientFactory api_factory: The api factory to use.
+    :param list[str] file_types: A list of the file_types for upload.
```

Comment:
is `file_types` csv/xls or the domain of things to be uploaded transaction/instrument/holding etc.

Reply:
Agreed `domain` is better.

I took file_type from the following but the name file_type makes more sense in the core cocoon code:

https://github.com/finbourne/lusid-python-tools/blob/master/lusidtools/cocoon/cocoon.py

Reply:
lets use domain here